EIPS/eip-7733.md

```diff
@@ -0,0 +1,114 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7742
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.